### PR TITLE
Change year dynamically in sphinx docs

### PR DESCRIPTION
**Pull request by kurianbenoy**
Issue context: https://github.com/fairlearn/fairlearn/issues/764#issuecomment-864426159

Signed-off-by: kurianbenoy-aot <kurian.benoy@aot-technologies.com>

### CODE REVIEW

**Review by hildeweerts — APPROVED** [review]
LGTM, thanks @kurianbenoy!

@fairlearn/fairlearn-maintainers if you don't have any comments feel free to merge this

File: docs/conf.py [hunk 1]
```diff
@@ -29,7 +30,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Fairlearn'
-copyright = '2021, Fairlearn contributors'
+copyright = f'{datetime.now().year}, Fairlearn contributors'
```
Comment (adrinjalali) [inline docs/conf.py:33]: should this not say `2018-2021, Fairlearn contributors`? I'm really not sure.

Comment (hildeweerts) [reply docs/conf.py:33]: I'm not sure whether it *has* to, but we could certainly add it (see [Miro's comment](https://github.com/fairlearn/fairlearn/pull/823#issuecomment-857661650)).

Comment (kurianbenoy) [reply docs/conf.py:33]: Yeah @adrinjalali. I will make that change

**Review by adrinjalali — APPROVED** [review]

**Pull request merged**